enterprise/frontend/src/metabase-enterprise/plugins.js

```diff
@@ -1,40 +1,73 @@
 import MetabaseSettings from "metabase/lib/settings";
 import { PLUGIN_IS_EE_BUILD } from "metabase/plugins";
 
+import { activateAdvancedPermissionsPlugin } from "./advanced_permissions";
+import { activateApplicationPermissionsPlugin } from "./application_permissions";
+import { activateAuditAppPlugin } from "./audit_app";
+import { activateAuthPlugin } from "./auth";
+import { activateCachingPlugin } from "./caching";
+import { activateCollectionCleanupPlugin } from "./clean_up";
+import { activateCollectionsPlugin } from "./collections";
+import { activateContentVerificationPlugin } from "./content_verification";
+import { activateEmailAllowListPlugin } from "./email_allow_list";
+import { activateEmailRestrictRecipientsPlugin } from "./email_restrict_recipients";
+import { activateEmbeddingPlugin } from "./embedding";
+import { activateFeatureLevelPermissionsPlugin } from "./feature_level_permissions";
+import { activateGroupManagersPlugin } from "./group_managers";
+import { activateHostingPlugin } from "./hosting";
+import { activateLicensePlugin } from "./license";
+import { activateLLMAutoDescriptionPlugin } from "./llm_autodescription";
+import { activateModelPersistencePlugin } from "./model_persistence";
+import { activateModerationPlugin } from "./moderation";
+import { enableResourceDownloadsPlugin } from "./resource_downloads";
+import { activateSandboxesPlugin } from "./sandboxes";
+import { activateSharedPlugin } from "./shared";
+import { activateSharingPlugin } from "./sharing";
+import { activateSnippetsPlugin } from "./snippets";
+import { activateToolsPlugin } from "./tools";
+import { activateTroubleshootingPlugin } from "./troubleshooting";
+import { activateUploadManagementPlugin } from "./upload_management";
+import { activateUserProvisioningPlugin } from "./user_provisioning";
+import { activateWhitelabelPlugins } from "./whitelabel";
+
 // SETTINGS OVERRIDES:
 
 // NOTE: temporarily use "latest" for Enterprise Edition docs
 MetabaseSettings.docsTag = () => "latest";
 PLUGIN_IS_EE_BUILD.isEEBuild = () => true;
 
-import "./shared";
-
-// PLUGINS:
-
-import "./hosting";
-import "./tools";
-import "./sandboxes";
-import "./auth";
-import "./caching";
-import "./collections";
-import "./content_verification";
-import "./whitelabel";
-import "./embedding";
-import "./snippets";
-import "./sharing";
-import "./moderation";
-import "./email_allow_list";
-import "./email_restrict_recipients";
-import "./advanced_permissions";
-import "./audit_app";
-import "./license";
-import "./model_persistence";
-import "./feature_level_permissions";
-import "./application_permissions";
-import "./group_managers";
-import "./llm_autodescription";
-import "./upload_management";
-import "./resource_downloads";
-import "./user_provisioning";
-import "./clean_up";
-import "./troubleshooting";
+let pluginEnabled = false;
+export const activateEEPlugins = () => {
+  if (pluginEnabled) {
```

Comment:
This is required because in unit tests we call `setupEnterprisePlugins` in `beforeEach` and some plugins can't be called twice (they add new elements to the admin settings etc), we could also move the check there.